src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -240,13 +250,16 @@ std::string renderSemWaitStats(void)
   jh.addFloat("subCache",          semTimeCacheGet());
   jh.addFloat("connectionContext", mutexTimeCCGet());
   jh.addFloat("timeStat",          semTimeTimeStatGet());
+  jh.addFloat("metrics",           ((float) metricsMgr.semWaitTimeGet()) / 1000000);
```

Comment:
Or maybe we thought it was not important enough... I don't remember.

Anyway, with regars to metrics API and its sems, NTC.